src/data-workspace/indicators-table-body/compute-indicator-value.js

```diff
@@ -83,13 +82,14 @@ const parseExpressionTemplate = (expression, values) => {
 
     return matches.reduce((substitudedExpression, match) => {
         const operand = match.replace(/[#{}]/g, '')
-        const { categoryOptionComboId } = parseFieldOperand(operand)
+        const { dataElementId, categoryOptionComboId } =
+            parseFieldOperand(operand)
         const value =
             (categoryOptionComboId
                 ? // For COCs we read the data directly from the form values
-                  getIn(values, operand)
+                  dataValues?.[dataElementId]?.[categoryOptionComboId]?.value
```

Comment:
Would it make sense to have a getter for this? There is already in the store, but that is "bound", but we could potentially extract it to a function, there's at least two places we do this deep "getter".

Reply:
I'm not sure if having a getter would improve things here as this is a function outside of a component/hook?